response_operations_ui/views/collection_exercise.py

```diff
@@ -105,7 +105,15 @@ def view_collection_exercise(short_name, period):
     ]
 
     ce_state = ce_details["collection_exercise"]["state"]
-    show_set_live_button = ce_state in ("READY_FOR_REVIEW", "FAILEDVALIDATION")
+    if ce_details["survey"]["surveyMode"] == "EQ":
```

Comment:
can you check why FAILEDVALIDATION is a part of this logic?

Reply:
It's been like that forever.  I think the theory of it is that it would fail validation, you'd fix whatever was wrong behind the scenes then you'd try again.

Whether that ever happened in practice is another story, but I don't think it's worth removing it here just to limit the amount of change in this PR.  We're already touching something quite important (the ability to set an exercise live) so I don't want to do too much

Reply:
So, Do I need to do something with FAILEDVALIDATION ?